core/config/project_settings.cpp

```diff
@@ -1619,6 +1619,50 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF_INTERNAL("internationalization/locale/translations_pot_files", PackedStringArray());
 	GLOBAL_DEF_INTERNAL("internationalization/locale/translation_add_builtin_strings_to_pot", false);
 
+	GLOBAL_DEF("navigation/world/map_use_async_iterations", true);
+
+	GLOBAL_DEF("navigation/avoidance/thread_model/avoidance_use_multiple_threads", true);
+	GLOBAL_DEF("navigation/avoidance/thread_model/avoidance_use_high_priority_threads", true);
+
+	GLOBAL_DEF("navigation/pathfinding/max_threads", 4);
+
+	GLOBAL_DEF("navigation/baking/use_crash_prevention_checks", true);
+	GLOBAL_DEF("navigation/baking/thread_model/baking_use_multiple_threads", true);
+	GLOBAL_DEF("navigation/baking/thread_model/baking_use_high_priority_threads", true);
+
+#ifdef DEBUG_ENABLED
+	GLOBAL_DEF("debug/shapes/navigation/edge_connection_color", Color(1.0, 0.0, 1.0, 1.0));
```

Comment:
As mentioned in dev chat we might want to use that opportunity and split the 2d and 3d debug projectsettings properties. It is often inconvenient to have the same values shared between them as the editor viewports have different colors, scale and backdrops between 2d and 3d.

Reply:
Looking at separating these, gotta see how to do it without losing existing data though will test

Reply:
See the `MOVE_PROJECT_SETTING` in https://github.com/godotengine/godot/pull/98528 for one way to do it.

Reply:
Thank you! Will take a look

Reply:
Done! Thank you that works well!